src/capymoa/stream/generator.py

```diff
@@ -330,43 +336,43 @@ def __init__(
 
         super().__init__(CLI=config_str, moa_stream=self.moa_stream)
 
-    def __str__(self):
-        attributes = [
-            (
-                f"instance_random_seed={self.instance_random_seed}"
-                if self.instance_random_seed != 1
-                else None
-            ),
-            (
-                f"number_of_classes={self.number_of_classes}"
-                if self.number_of_classes != 2
-                else None
-            ),
-            (
-                f"number_of_attributes={self.number_of_attributes}"
-                if self.number_of_attributes != 10
-                else None
-            ),
-            (
-                f"number_of_drifting_attributes={self.number_of_drifting_attributes}"
-                if self.number_of_drifting_attributes != 2
-                else None
-            ),
-            (
-                f"magnitude_of_change={self.magnitude_of_change}"
-                if self.magnitude_of_change != 0.0
-                else None
-            ),
-            (
-                f"noise_percentage={self.noise_percentage}"
-                if self.noise_percentage != 5
-                else None
-            ),
-            (
-                f"sigma_percentage={self.sigma_percentage}"
-                if self.sigma_percentage != 10
-                else None
-            ),
-        ]
-        non_default_attributes = [attr for attr in attributes if attr is not None]
-        return f"HyperPlaneRegression({', '.join(non_default_attributes)})"
+    # def __str__(self):
```

Comment:
Most of the attributes : self.instance_random_seed, self.number_of_classes, self.number_of_drifting_attributes, etc are not set.
When one calls __str__ it fails